plugin/src/helpers/config.ts

```diff
@@ -1,10 +1,20 @@
+import type { NetlifyConfig } from '@netlify/build'
 import { readJSON, writeJSON } from 'fs-extra'
+import type { Header } from 'next/dist/lib/load-custom-routes'
 import type { NextConfigComplete } from 'next/dist/server/config-shared'
 import { join, dirname, relative } from 'pathe'
 import slash from 'slash'
 
 import { HANDLER_FUNCTION_NAME, ODB_FUNCTION_NAME } from '../constants'
 
+const ROUTES_MANIFEST_FILE = 'routes-manifest.json'
+
+type NetlifyHeaders = NetlifyConfig['headers']
+
+// This is the minimal amount of typing required for now
+// add other properties as needed from the routes-manifest.json
+export type RoutesManifest = { headers: Header[] }
```

Comment:
[pebble] You can use [the full type](https://github.com/netlify/netlify-plugin-nextjs/blob/main/plugin/src/helpers/types.ts#L42)

Reply:
Didn't realize there was already a full type defined already. I'll definitely use that instead.